package.json

```diff
@@ -71,7 +71,8 @@
     "font-manager": "https://github.com/stream-labs/font-manager/releases/download/v1.0.4/iojs-v3.0.1-font-manager.tar.gz",
     "node-fontinfo": "https://github.com/stream-labs/node-fontinfo/releases/download/v0.0.7/iojs-v3.0.1-node-fontinfo.tar.gz",
     "node-libuiohook": "https://github.com/stream-labs/node-libuiohook/releases/download/v0.0.4/iojs-v3.0.1-node-libuiohook.tar.gz",
-    "obs-studio-node": "https://github.com/stream-labs/obs-studio-node/releases/download/v0.3.26/iojs-v3.0.0-signed.tar.gz",
+    "obs-studio-node": "https://github.com/stream-labs/obs-studio-node/releases/download/v0.3.26/iojs-v3.0.0-signed.tar.gz",    
+    "crash-handler": "https://github.com/stream-labs/crash-handler/releases/download/v0.0.4/iojs-v3.0.4-crash-handler.tar.gz",
```

Comment:
This should probably be added alphabetically to stay consistent with the rest of the packages.